synapse/handlers/room_member.py

```diff
@@ -547,6 +547,13 @@ def copy_user_state_on_room_upgrade(self, old_room_id, new_room_id, user_ids):
                     old_room_id, new_room_id, user_id
                 )
             except Exception:
+                logger.warning(
+                    "Error copying tags and/or push rules from rooms %s to %s for user %s. "
+                    "Skipping...",
```

Comment:
we still need to say what the exception is. you've got three choices:

* `except Exception as e` and `logger.warning("words words words: %s", e)`
* `logger.warning("words words words", exc_info=1)`, which will log a stacktrace too. Don't do this unless the exception suggests a bug in synapse.
* `logger.exception("words words words")`, which is the same as `logger.error(..., exc_info=1)`